.stylelintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
this should be a .js file
https://stylelint.io/user-guide/configure

Looks like default is _stylelint.config.js_

Reply:
every .js file I have tried fails. no extension and .json work
``` error  Parsing error: Unexpected token, expected ";"```

Reply:
`.stylelintrc` has worked for me in multiple projects

Reply:
Ah, interesting.  I'll give it a try, as it seems per their docs it "should" work.  🤷‍♂️ 

Not a deal breaker if not, but would be nice to keep all our config files under lint as well.